ffi/src/expressions.rs

```diff
@@ -146,7 +146,9 @@ fn visit_expression_column_impl(
     state: &mut KernelExpressionVisitorState,
     name: DeltaResult<String>,
 ) -> DeltaResult<usize> {
-    Ok(wrap_expression(state, Expression::Column(name?)))
+    // TODO: FIXME: This is incorrect if any field name in the column path contains a period.
+    let name = ColumnName::new(name?.split('.')).into();
```

Comment:
create follow-up issue?

Reply:
This is not worse than before, just documented now. And it's already all under the umbrella of https://github.com/delta-incubator/delta-kernel-rs/issues/86, and the next PR will fix it (plus the remaining gaps elsewhere).

Reply:
Created an issue for this PR to close, and added it as a task under the umbrella issue.

Reply:
great thanks :)